src/main/java/com/target/studentinfo/dto/mapper/StudentMapper.java

```diff
@@ -0,0 +1,15 @@
+package com.target.studentinfo.dto.mapper;
+
+import com.target.studentinfo.dto.request.StudentRequest;
+import com.target.studentinfo.dto.response.StudentResponse;
+import com.target.studentinfo.model.Student;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface StudentMapper {
+    StudentMapper MAPPER = Mappers.getMapper(StudentMapper.class);
+    Student fromReqToModel(StudentRequest studentRequest);
+    StudentResponse fromModelToResponse(Student student);
+
+}
```

Comment:
Use a manual mapper instead of mapper library for visibility and flexibility to update it as and when we want.

```
public class StudentMapperNew {

    public Student toStudent(StudentRequest studentRequest) {
        return Student.builder()
                .firstName(studentRequest.getFirstName())
                .lastName(studentRequest.getLastName())
                .build();
    }

    public StudentResponse toStudentResponse(Student student) {
        return StudentResponse.builder()
                .firstName(student.getFirstName())
                .lastName(student.getLastName())
                .build();
    }
}
```
Use `@AllArgsConstructor` and `@Builder` for Student and StudentResponse.